lib/android-helpers.js

```diff
@@ -506,7 +506,12 @@ helpers.pushStrings = async function (language, adb, opts) {
   // clean up remote string.json if present
   await adb.rimraf(remoteFile);
 
-  const app = opts.app || await adb.pullApk(opts.appPackage, opts.tmpDir);
+  let app = '';
```

Comment:
do not assign app to any value by default

Reply:
ah... `fs.exists` in L511 is https://github.com/appium/appium-support/blob/42a2ab3f605d3cd3f78fe26a609c95790fbab963/lib/fs.js#L23 
👍